module/VuFind/src/VuFind/View/Helper/Root/MakeTag.php

```diff
@@ -94,6 +95,7 @@ class MakeTag extends \Laminas\View\Helper\AbstractHelper
         'ins',
         'kbd',
         'label',
+        'li',
```

Comment:
@crhallberg, I think I see what's going on here -- the list of "valid body tags" you found is literally "tags that are allowed as direct children of `<body>`". But you can't put a `<li>` in there because it needs to be contained by a list. Similarly, I notice that this list is missing things like `<tr>` and `<td>`. I'm sure there are others! It's probably worth opening a separate PR to address this and expand the list. What do you think?

Reply:
https://github.com/vufind-org/vufind/pull/2560

Reply:
Thanks! I've merged the new PR and reconciled the conflicts here. Much better now!